packages/documents/lib/meta.js

```diff
@@ -74,6 +81,21 @@ const getAudioSource = doc => {
   return audioSource
 }
 
+const getTotalMediaDurationMinutes = doc => {
+  let total = 0
+  if (doc.meta && doc.meta.audioSource && doc.meta.audioSource.durationMs) {
+    total += doc.meta.audioSource.durationMs
+  }
+  const ids = {}
+  visit(doc.content, 'zone', (node, i, parent) => {
+    if (node.identifier === 'EMBEDVIDEO' && node.data.durationMs && !ids[node.data.id]) {
```

Comment:
`node.data` might be undefined and calling `node.data.durationMs` will throw an error.